### ISSUE TITLE
Error when executed "npm start"

### ISSUE THREAD

**Issue by edbek** [davidkpiano/frontend-masters-xstate-workshop#4]
When executed "npm start" the following pops up:

ENOENT: no such file or directory, open '...\testem.js'

**Comment by camidoubts** [comment]
I couldn't make it either. In the first video David says that you only have to save one file for it to work, but other errors appear when you do that.

**Comment by iguanus** [comment]
Same for me. Which OS are you using? 

Windows 10, machine complains about testem.js not existing at the root of the project.
ArchLinux was able to run it without issues.
Ubuntu (virtual machine on C9) also installs and runs without issues.

So it seems to be OS-dependent.

**Comment by camidoubts** [comment]
hi, i'm on windows 10. it worked for me by running `parcel **/*.html` directly instead of `npm start`

**Comment by GiancarlosIO** [comment]
Same here 😢 
The first time I get this error:
```
ENOENT: no such file or directory, open 'D:\frontend-projects\state-machines-in-javascript-with-xstate\frontend-masters-xstate-workshop\testem.js'
```

Then, after I save a HTML file I get:
The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received null

**Comment by GiancarlosIO** [comment]
Alright, I just found what was the error.
I have created a PR to fix it: https://github.com/davidkpiano/frontend-masters-xstate-workshop/pull/6

**Comment by davidkpiano** [comment]
Fixed in #6, thanks!
